docs/send-data/opentelemetry-collector/use-case-prometheus.md

```diff
@@ -0,0 +1,156 @@
+# HELP mock_metric_counter_2
+# TYPE mock_metric_counter_2 counter
+mock_metric_counter_2{app="prometheus-mock-metrics-server",client_addr="client2"} 5362
+# HELP mock_metric_gauges_0
+# TYPE mock_metric_gauges_0 gauge
+mock_metric_gauges_0{app="prometheus-mock-metrics-server",client_addr="client1"} 71
+# HELP mock_metric_gauges_1
+# TYPE mock_metric_gauges_1 gauge
+mock_metric_gauges_1{app="prometheus-mock-metrics-server",client_addr="client2"} 44
+# HELP mock_metric_gauges_2
+# TYPE mock_metric_gauges_2 gauge
+mock_metric_gauges_2{app="prometheus-mock-metrics-server",client_addr="client3"} 4
+# HELP mock_metric_gauges_3
+# TYPE mock_metric_gauges_3 gauge
+mock_metric_gauges_3{app="prometheus-mock-metrics-server",client_addr="client3"} 41
+# HELP mock_metric_request_total
+# TYPE mock_metric_request_total counter
+mock_metric_request_total{app="prometheus-mock-metrics-server"} 0
+```
+
+## Step 2: Add Prometheus receiver to your OpenTelemetry Collector configuration
+
+Next, we'll configure the [Sumo Logic OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) to receive metrics from the Prometheus Metrics Server Simulator, and then forward them to Sumo Logic.
+
+1. [Install the Sumo Logic OpenTelemetry Collector](/docs/send-data/opentelemetry-collector/).
+2. Create a new configuration file for the OpenTelemetry Collector with the following content:
+   ```yml title="/etc/otelcol-sumo/conf.d/prom.yaml"
+   receivers:
+     prometheus:
+       config:
+         scrape_configs:
+           - job_name: 'mock-metrics'
+             scrape_interval: 10s
+             static_configs:
+               - targets: ['localhost:8080']
+   service:
+     pipelines:
+       metrics:
+         receivers: [prometheus]
+         exporters: [sumologic]
+   ```
+  * In the above configuration, we define a `prometheus` receiver that scrapes metrics from the Prometheus Metrics Server Simulator running on `localhost:8080`. We then define a metrics pipeline that uses the `prometheus` receiver to scrape the metrics and send them to Sumo Logic via the `sumologic` exporter.
+3. Save the configuration file as `/etc/otelcol-sumo/conf.d/prom.yaml`.
+4. Start the OpenTelemetry Collector with the following command.
+   ```bash
+   sudo otelcol-sumo \
+   --config=/etc/otelcol-sumo/sumologic.yaml \
+   --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+   ```
+
+This will start the collector with the configuration you defined, and begin receiving and exporting metrics to Sumo Logic.
+
+
+## Step 3: Create a New Dashboard in Sumo Logic
+
+Finally, we will create a new dashboard in Sumo Logic to visualize the custom Prometheus-formatted metrics we are receiving.
+
+To create a new dashboard:
+
+1. Go to the Sumo Logic **Home** tab, click **+ New**, then select **Dashboard (New)** from the dropdown list.<br/><img src={useBaseUrl('img/dashboards-new/create-dashboard-new/dashboard-new-new.png')} alt="dashboard-new-new.png" width="500"/>
```

Comment:
@kimsauce, now that all articles have hidden Kanso instructions, you need to add them to this draft. For example:

`<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  Click the **+ New** button at the top of the screen and select **Dashboard**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Dashboards > New Dashboard**. You can also click the **Go To...** menu at the top of the screen and select **New Dashboard**.  Kanso-->`